app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt

```diff
@@ -38,6 +38,8 @@ class AdminPinActivityPresenter @Inject constructor(
     activity.title = activity.getString(R.string.add_profile_title)
     activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
     activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
+    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
+
```

Comment:
Delete the blank lines here to pass the test

Reply:
+1